code/drasil-example/gamephysics/lib/Drasil/GamePhysics/GenDefs.hs

```diff
@@ -64,16 +65,28 @@ conservationOfMomentDeriv = foldlSent [S "When bodies collide, they exert",
 --------------------------Acceleration due to gravity----------------------------
 accelGravityGD :: GenDefn
 accelGravityGD = gd (equationalModel' accelGravityQD) (getUnit QP.acceleration) (Just accelGravityDeriv)
-   [dRef accelGravitySrc] "accelGravity" [accelGravityDesc]
+   [dRef accelGravitySrc] "accelGravity" accelGravityDesc
 
 accelGravityQD :: ModelQDef
 accelGravityQD = mkQuantDef' QP.gravitationalAccel (nounPhraseSP "Acceleration due to gravity") accelGravityExpr
 
-accelGravityDesc :: Sentence
-accelGravityDesc = foldlSent [S "If one of the", plural QPP.mass `S.is` S "much larger than the other" `sC`
-  (S "it is convenient to define a gravitational field around the larger mass as shown above" !.),
-  S "The negative sign" `S.inThe` S "equation indicates that the", phrase QP.force, S "is an attractive",
-  phrase QP.force]
+accelGravityDesc :: [Sentence]
+accelGravityDesc =
+  [ foldlSent
+      [ (S "If one of the" +:+ plural QPP.mass) `sC` S "much larger than the other"
```

Comment:
you removed the `S.is` (and replaced it with a comma, i.e. `sC`) which makes the resulting sentence non-grammatical.